### PR TITLE
Replace 1C Enterprise grammar

### PR DESCRIPTION
**Pull request by nixel2007**
Replace source repo for bsl (1C Enterprise) language

## Description

Since atom was sunset and atom plugin is not maintained anymore, let's get grammar from vscode plugin. 

## Checklist:

- [x] **I am changing the source of a syntax highlighting grammar**
  - Old: https://github.com/1c-syntax/atom-language-1c-bsl
  - New: https://github.com/1c-syntax/vsc-language-1c-bsl

### CODE REVIEW

**Review by lildude — CHANGES_REQUESTED** [review]
I'm not sure how you did it, but we only need the cached license from the new grammar. Please remove the others.

**Comment by nixel2007** [comment]
@lildude yeah, they came from submodule let me change it.

File: vendor/licenses/git_submodule/vsc-language-1c-bsl.dep.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-name: atom-language-1c-bsl
-version: 289454b53797ddc6fe193486c5a08310c214da7e
+name: vsc-language-1c-bsl
+version: 10e5889362637b4e46f5d9bcd9d860b124b660b5
 type: git_submodule
-homepage: https://github.com/xDrivenDevelopment/atom-language-1c-bsl.git
-license: mit
+homepage: https://github.com/1c-syntax/vsc-language-1c-bsl.git
+license: other
```
Comment (lildude) [inline vendor/licenses/git_submodule/vsc-language-1c-bsl.dep.yml:6]:
```suggestion
license: mit
```

**Comment by nixel2007** [comment]
@lildude done

**Comment by nixel2007** [comment]
Kindly ping @lildude

**Review by lildude — APPROVED** [review]
LGTM. Thanks.

As an aside, there's really no need for the repeated pings. Why? 👇

Note: this PR will not be merged until close to when the next release is made. See [here](https://github.com/github/linguist/blob/master/docs/troubleshooting.md#my-linguist-pr-has-been-merged-but-github-doesnt-reflect-my-changes) for more details.

**Comment by lildude** [comment]
> How could I test, if new format of grammar works correctly? Thanks in advance.

Replace the grammar with itself like you did when replacing the old grammar with the new.

**Review by lildude — APPROVED** [review]

**Pull request merged**